templates/Inspector.ss

```diff
@@ -29,7 +29,7 @@
             <h2>Cache status: (Cache / DB)</h2>
             <ul>
                 <% loop $StatusList %>
-                <li><code>$Status</code></li>
+                <li><code>$Status<% if $Missed %>. IDs that Caches missed: $Missed<% end_if %></code></li>
```

Comment:
No, the IDs are what we want to see as devs (the feature is mostly used by devs other than end-users), beside there are so many occasions that the page title might be same, such as "Embassy" pages in MFAT, where they all titled as "Embass" only uses the different location attribute to differentiate . :)